uint32 action
---
bool success
string message
